nats-server-config-reloader.yaml

```diff
@@ -0,0 +1,108 @@
+package:
+  name: nats-server-config-reloader
+  version: "0.17.1"
+  epoch: 0
+  description: "NATS server configuration reloader utility"
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/nats-io/nack.git
+      tag: v${{package.version}}
+      expected-commit: 03e5e930036a5baacc12ef604cd57ad6ff6ada96
+
+  - uses: go/build
+    with:
+      packages: ./cmd/nats-server-config-reloader
+      output: nats-server-config-reloader
+      ldflags: -X main.BuildTime=$(date -u +%Y-%m-%dT%H:%M:%S%z) -X main.GitInfo=$(git rev-parse --abbrev-ref HEAD)-$(git rev-parse --short HEAD)$(git diff --quiet || echo "-dirty") -X main.Version=${{package.version}}
+
+  - runs: |
+      mkdir -p ${{targets.destdir}}/usr/local/bin
+      cp -r ${{targets.destdir}}/usr/bin/nats-server-config-reloader ${{targets.destdir}}/usr/local/bin/nats-server-config-reloader
+      chmod 755 ${{targets.destdir}}/usr/local/bin/nats-server-config-reloader
+      cp cicd/assets/entrypoint.sh ${{targets.destdir}}/entrypoint.sh
```

Comment:
if this file is going to be the entrypoint of the image we should create a sub-package like ${{package.name}}-oci-entrypoint, you can search through Wolfi os repository for the -oci-entrypoint to see other examples of how to do that.